specifyweb/frontend/js_src/lib/components/Preferences/UserDefinitions.tsx

```diff
@@ -1936,6 +1937,40 @@ export const userPreferenceDefinitions = {
       },
     },
   },
+  batchEdit: {
+    title: batchEditText.batchEdit(),
+    subCategories: {
+      query: {
```

Comment:
(optional) if this limit is so easily changeable by the user, instead of being a limit, should we just display a warning when someone tries to batch edit more records than this?

(optional) and then we might not even need a pref for this as at that point we could just have a `const batchEditWarningSize = 5_000;`.